### PR TITLE
Test post join us api

### PR DESCRIPTION
**Pull request by hazemessam**
## Issue that this pull request solves

 Ref issue - # 710 

### Brief description of what is fixed or changed
- Removed 2 duplicated fields [domain, department] from join us validation schema
-  Added test for `POST /joinus`
## Types of changes

_Put an `x` in the boxes that apply_

- [x] Bugfix (non-breaking change which fixes an issue)
- [x] New feature (non-breaking change which adds functionality)
- [ ] Breaking change (fix or feature that would cause existing functionality to not work as expected)
- [ ] Documentation update (Documentation content changed)
- [ ] Other (please describe): 

## Checklist

_Put an `x` in the boxes that apply_

- [x] My code follows the style guidelines of this project
- [x] I have performed a self-review of my own code
- [x] I have commented my code, particularly in hard-to-understand areas
- [ ] I have made corresponding changes to the documentation
- [x] My changes generate no new warnings
- [x] My changes does not break the current system and it passes all the current test cases.

## Screenshots
image

### CODE REVIEW

File: backend/app/routes/joinUs/@validationSchema/index.js [hunk 1]
```diff
@@ -5,8 +5,6 @@ const JoinUsValidationSchema = Joi.object().keys({
   email: Joi.string().required(),
   linkdin: Joi.string().required(),
   description: Joi.string().required(),
-  domain: Joi.array().required(),
```
Comment (Kajol-Kumari) [inline backend/app/routes/joinUs/@validationSchema/index.js:8]: This is different from the `otherDomain` field so please don't remove it.

**Review by Kajol-Kumari — CHANGES_REQUESTED** [review]

File: backend/app/routes/joinUs/@validationSchema/index.js (hunk 1 above)
Comment (hazemessam) [reply backend/app/routes/joinUs/@validationSchema/index.js:8]: `domain` duplicates `interestedDomain`

Comment (Kajol-Kumari) [reply backend/app/routes/joinUs/@validationSchema/index.js:8]: Ah, correct! I misse this field :)

**Comment by hazemessam** [comment]
@Kajol-Kumari Is still any problem with this PR?

**Review by Kajol-Kumari — APPROVED** [review]
LGTM 👍

**Pull request merged**